go/src/koding/kites/kloud/provider/aws/default.json

```diff
@@ -0,0 +1,20 @@
+{
+  "provider": {
+    "aws": {
+      "access_key": "${var.aws_access_key}",
+      "secret_key": "${var.aws_secret_key}"
+    }
+  },
+  "resource": {
+    "aws_instance": {
+      "aws-instance": {
+        "ami": "",
+        "instance_type": "${var.userInput_instance_type}",
```

Comment:
this should remain as in client example, it shouldn't ask anything by default. let's replace these `userInput`'s to something fixed like in client folder.